packages/stack/src/TransitionConfigs/HeaderStyleInterpolators.tsx

```diff
@@ -101,8 +101,8 @@ export function forFade({
 }: StackHeaderInterpolationProps): StackHeaderInterpolatedStyle {
   const progress = add(current.progress, next ? next.progress : 0);
   const opacity = interpolate(progress, {
-    inputRange: [0, 1, 2],
-    outputRange: [0, 1, 0],
+    inputRange: [0, 1, 1.9999, 2],
```

Comment:
What's the reason to have `1.9999` instead of changing `0` to `1` in the `outputRange`?

Reply:
It is to hide the screen when the transition is finished. I can remove it if it's not necessary.